language/bytecode_verifier/tests/code_unit_tests.rs

```diff
@@ -0,0 +1,48 @@
+use bytecode_verifier::CodeUnitVerifier;
```

Comment:
Do we plan on extending this to check for different kinds of errors? (besides just InvalidFallThrough)

If yes, do we plan on sticking all of these in this one file? Or separating into different files?

Reply:
`CodeUnitVerifier` actually reports a variety of errors (it is what runs the reference safety and type safety analysis), but there were no unit tests for it before. I could imagine adding more unit tests for errors that are hard to check with functional tests in this file.